cpg-language-typescript/build.gradle.kts

```diff
@@ -37,23 +38,74 @@ mavenPublishing {
     }
 }
 
-node {
-    download.set(true)
-    version.set(libs.versions.node)
-    nodeProjectDir.set(file("${project.projectDir.resolve("src/main/nodejs")}"))
-}
+val compileMacOSx8664 =
+    tasks.register<RunDenoTask>("compileMacOSAmd64") {
+        dependsOn(tasks.installDeno)
+        command(
+            "compile",
+            "-E",
+            "-R",
+            "--target",
+            "x86_64-apple-darwin",
+            "-o",
+            "build/resources/main/typescript/parser-macos-amd64",
+            "src/main/typescript/src/parser.ts",
+        )
+        outputs.dir("build/resources/main/typescript")
+        outputs.cacheIf { true }
+    }
+
+val compileMacOSArm64 =
+    tasks.register<RunDenoTask>("compileMacOSArm64") {
+        dependsOn(tasks.installDeno)
+        command(
+            "compile",
+            "-E",
+            "-R",
+            "--target",
+            "aarch64-apple-darwin",
+            "-o",
+            "build/resources/main/typescript/parser-macos-arm64",
+            "src/main/typescript/src/parser.ts",
+        )
+        outputs.dir("build/resources/main/typescript")
+        outputs.cacheIf { true }
+    }
 
-val pnpmBuild by
-    tasks.registering(PnpmTask::class) {
-        inputs.file("src/main/nodejs/package.json").withPathSensitivity(PathSensitivity.RELATIVE)
-        inputs.file("src/main/nodejs/pnpm-lock.yaml").withPathSensitivity(PathSensitivity.RELATIVE)
-        inputs.dir("src/main/nodejs/src").withPathSensitivity(PathSensitivity.RELATIVE)
-        outputs.dir("build/resources/main/nodejs")
+val compileLinuxX8664 =
+    tasks.register<RunDenoTask>("compileLinuxAmd64") {
+        dependsOn(tasks.installDeno)
+        command(
+            "compile",
+            "-E",
+            "-R",
+            "--target",
+            "x86_64-unknown-linux-gnu",
+            "-o",
+            "build/resources/main/typescript/parser-linux-amd64",
+            "src/main/typescript/src/parser.ts",
+        )
+        outputs.dir("build/resources/main/typescript")
         outputs.cacheIf { true }
+    }
 
-        workingDir.set(file("src/main/nodejs"))
-        pnpmCommand.set(listOf("run", "bundle"))
-        dependsOn(tasks.getByName("pnpmInstall"))
+val compileLinuxArm64 =
```

Comment:
So, there is no windows support?

Reply:
Ah there is windows support (see https://docs.deno.com/runtime/reference/cli/compile/#supported-targets), so I can add that